src/components/pages/PendingApproval/PendingApproval.js

```diff
@@ -19,7 +19,7 @@ const PendingApproval = props => {
 
   const [toggle, setToggle] = useState(false);
 
-  useEffect(() => {
+  useLayoutEffect(() => {
```

Comment:
I'm loving the use of useLayoutEffect per the Stakeholder's suggestion, great job